README.md

```diff
@@ -4,83 +4,112 @@ We are committed to providing an open platform for developers to build upon.
 
 While the Cards themselves are stored on the Ethereum blockchain (or in the Starkware rollup) we support an API that provides more detailed information.
 
-## GraphQL
-
-Our primary API is specified in the [GraphQL](https://graphql.org/) query language and documented in our [GraphQL playground](https://api.sorare.com/graphql/playground).
+The Sorare API is provided by [GraphQL](https://graphql.org/). Documentation can be found under the Docs section in the [GraphQL playground](https://api.sorare.com/graphql/playground).
 
 ## Authentication
```

Comment:
May want to call this User Authentication, which contrasts to OAuth Authentication or API Key authentication. Where do we talk about getting an API key? We just say "on demand" I think.

Reply:
Ah yes, good idea.